packages/core/sdk/src/auth/internal/CoreStitchAuth.ts

```diff
@@ -458,13 +463,20 @@ export default abstract class CoreStitchAuth<TStitchUser extends CoreStitchUser>
    * Processes the response of the login/link request, setting the authentication state if appropriate, and
    * requesting the user profile in a separate request.
    */
-  private processLoginResponse(
+  private processLogin(
```

Comment:
In addition to that, we should actually pass a response here and we should instead of creating an auth info in the credential, have the response body marshaled.

Reply:
Perhaps a better solution is to have this method stay as `processLogin` but have it accept an `ApiAuthInfo` or `AuthInfo` instead of a `Response`, and move the response decoding logic to a method called `processLoginResponse` which accepts a non-optional `Response`.   That way, the codepaths for the non-redirect logins can stay the same, whereas the link code path can call `processLogin` directly with the auth info it reconstructs from the URL. The credential would then only need to hold the provider type and name. Thoughts @edaniels ?

Reply:
👍 handled offline